Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1291,21 +1294,17 @@ def resolve_detection_command():
     """
     args = demisto.args()
     ids = argToList(args.get('ids'))
-    usernames = argToList(args.get('username'))
-    if usernames and ids:
-        raise ValueError('Only one of the arguments ids or username should be provided, not both.')
-    if not usernames and not ids:
-        raise ValueError('One of the arguments ids or username must be provided, none given.')
-    if usernames:
-        ids = []
-        for username in usernames:
-            username_uuid = get_username_uuid(username)
-            ids.append(username_uuid)
+    username = args.get('username')
```

Comment:
why did we stop supporting a list of users?

Reply:
In the old code the uuid was added to the detection ids list instead of assigning it's value to the `assigned_to_uuid` parameter.
The api doesnt support more than one user to assign the detections to.
<img width="724" alt="Screen Shot 2020-09-22 at 19 23 36" src="https://user-images.githubusercontent.com/21142167/93910013-2e354f00-fd09-11ea-9878-abe3cc6f7678.png">